lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.pm

```diff
@@ -0,0 +1,790 @@
+            is_output => 1,
+        },
+
+        old_sample => {
+            is => "Text",
+            doc => "Sample name in the VCF file to use",
+        },
+
+        new_sample => {
+            is => "Text",
+            doc => "Sample name to call the old sample during comparisons",
+        },
+
+        gold_vcf => {
+            is => "Text",
+            doc => "VCF file containing gold standard variant sites "
+                   . "to be used to measure the evaluation VCF"
+        },
+
+        gold_sample => {
+            is => "Text",
+            doc => "Sample name in gold standard vcf file",
+        },
+
+        roi => {
+            is => "Text",
+            doc => "BED file of the target regions to restrict the analysis to",
+        },
+
+        true_negative_bed => {
+            is => "Text",
+            doc => "BED file containing regions where no "
+                   . "variant call should be made",
+        },
+
+        true_negative_size => {
+            is => "Integer",
+            doc => "Use this number as the size of the TN BED "
+                   . "file rather than calculating on the fly",
+            is_optional => '1',
+        },
+
+        pass_only_expression => {
+            is => "Text",
+            doc => "String to pass to vcflib vcffilter in order to "
+                   . "select variants from the evaluation VCF",
+            default_value => q{-g 'FT = PASS | FT = .'},
+        },
+
+        clean_indels => {
+            is => "Text",
+            doc => "Whether or not to exclude indels that don't "
+                   . "actually overlap the ROI",
+            default_value => 0,
+        },
+    ],
+
+    has_output => [
+        rawstats => {
+            is => "HASH",
+            doc => "The raw stats generated during primary execution",
+        },
+    ],
+};
+
+sub execute {
+    my $self = shift;
+
+    my $vcf = $self->vcf;
+    my $roi = $self->roi;
+    my $gold_vcf = $self->gold_vcf;
+    my $tn_bed = $self->true_negative_bed;
+    my $old_sample = $self->old_sample;
+    my $new_sample = $self->new_sample;
+    my $gold_sample = $self->gold_sample;
+    my $clean_indels = $self->clean_indels;
+
+    my $output_dir = $self->output_directory;
+
+    die "Output dir $output_dir does not exist" unless -d $output_dir;
+    my $orig_file = "$output_dir/orig.vcf";
+    my $input_file = "$output_dir/clean.vcf";
+    my $final_input_file = "$output_dir/final.vcf";
+    my $final_gold_file = "$output_dir/gold-final.vcf";
+    my $final_tn_file = "$output_dir/tn-final.bed";
+    my $variants_dir = "$output_dir/variants";
+    my $fp_roi_file = "$output_dir/fp_in_roi.vcf";
+    my $compare_file = "$output_dir/compare.txt";
+
+    Genome::Sys->create_directory($variants_dir);
+    Genome::Sys->create_symlink($vcf, "$output_dir/orig.vcf");
+    Genome::Sys->create_symlink($roi, "$output_dir/roi.bed");
+    Genome::Sys->create_symlink($gold_vcf, "$output_dir/gold.vcf");
```

Comment:
The symlink destinations here should probably refer to the file variables just declared above.